ToolkitApp/tabs/src/components/sample/TabConstants.jsx

```diff
@@ -4,30 +4,46 @@ export const RSC_DOCUMENTATION_URL = "https://raw.githubusercontent.com/Microsof
 export const OFFICIAL_RSC_URL = "https://docs.microsoft.com/en-us/microsoftteams/platform/graph-api/rsc/resource-specific-consent#:~:text=Resource-specific%20consent%20%28RSC%29%20is%20a%20Microsoft%20Teams%20and,manage%20specific%20resources%E2%80%94either%20teams%20or%20chats%E2%80%94within%20an%20organization.";
 export const README_HEADER = "---";
 export const MS_GRAPH_DOCS = "(https://docs.microsoft.com/en-us/";
+export const GRAPH_URL = "https://graph.microsoft.com/"
 //TO DO: Support multi languages. Attached to ADO ticket #38594
 
 // This list of RSC permissions is copied directly from `webApplicationInfo.applicationPermissions` in "../../../../teamsApp/manifest.json"
-export const RSC_LIST = [   "TeamSettings.Read.Group",
-                            "ChannelMessage.Read.Group",
-                            "TeamSettings.Edit.Group",
-                            "ChannelSettings.ReadWrite.Group",
-                            "Channel.Create.Group",
-                            "Channel.Delete.Group",
-                            "TeamsApp.Read.Group",
-                            "TeamsTab.Read.Group",
-                            "TeamsTab.Create.Group",
-                            "TeamsTab.ReadWrite.Group",
-                            "TeamsTab.Delete.Group",
-                            "Member.Read.Group",
-                            "Owner.Read.Group",
-                            "ChatSettings.Read.Chat",
-                            "ChatSettings.ReadWrite.Chat",
-                            "ChatMessage.Read.Chat",
-                            "ChatMember.Read.Chat",
-                            "Chat.Manage.Chat",
-                            "TeamsTab.Read.Chat",
-                            "TeamsTab.Create.Chat",
-                            "TeamsTab.Delete.Chat",
-                            "TeamsTab.ReadWrite.Chat",
-                            "TeamsAppInstallation.Read.Chat",
-                            "OnlineMeeting.ReadBasic.Chat"]
\ No newline at end of file
+export const RSC_LIST = [
+    "TeamSettings.Read.Group",
+    "ChannelMessage.Read.Group",
+    "TeamSettings.Edit.Group",
+    "ChannelSettings.ReadWrite.Group",
+    "Channel.Create.Group",
+    "Channel.Delete.Group",
+    "TeamsApp.Read.Group",
+    "TeamsTab.Read.Group",
+    "TeamsTab.Create.Group",
+    "TeamsTab.ReadWrite.Group",
+    "TeamsTab.Delete.Group",
+    "Member.Read.Group",
+    "Owner.Read.Group",
+    "ChatSettings.Read.Chat",
+    "ChatSettings.ReadWrite.Chat",
+    "ChatMessage.Read.Chat",
+    "ChatMember.Read.Chat",
+    "Chat.Manage.Chat",
+    "TeamsTab.Read.Chat",
+    "TeamsTab.Create.Chat",
+    "TeamsTab.Delete.Chat",
+    "TeamsTab.ReadWrite.Chat",
+    "TeamsAppInstallation.Read.Chat",
+    "OnlineMeeting.ReadBasic.Chat"
+];
+
+export const requestTypes = [
```

Comment:
can change it to an enum

Reply:
JavaScript doesn't support enums, would jsx support enums?

Reply:
emm I see it's JS not TS. probably we could try something like 
```
const requestTypes = {
   POST = "POST"
   ...
}
```
The intention is to use a map instead of a list because I see your code is using something like `requestTypes[0]` to get the value. Using list will be error prone